### ISSUE TITLE
Add a page for officers to view all current inductee information

### ISSUE THREAD

**Issue by thai-tgil** [HKN-UCSD/hkn-server#258]
As title. This page needs to have a table component of some sort to display the information of each inductee in columns, including the new information we are adding to the account sign up form (check issue #255 for more information). Refer to Inductee Points page in `frontend/src/pages/InducteePointsPage` for an example of how to do this. Furthermore, you'll need to have a drop down menu to select which columns are needed.

You might need to do something with `getMultipleUsers()` handler in `UserController` in backend or do something with the method calls it makes to get only inductees in the current induction class. `buildMultipleUserQuery` is a good place to start (read up on TypeORM's docs for this). 

Go through requirements for this in the planning document for more details and ask questions if you have any.

Given our time frame and what we have available, we probably cannot get to allowing officers to see inductee information from previous induction cycles yet so let's not worry about that for now in this PR. Probably can put this in a separate issue later on.

This is possibly (most likely) blocked by #255

**Comment by thai-truong** [comment]
This might already be covered by my PR on induction cycle CRUD... Sorry about that, we can probably give you another task to do soon, let's discuss this at the meeting

**Comment by kyle1373** [comment]
> This might already be covered by my PR on induction class CRUD... Sorry about that, we can probably give you another task to do soon, let's discuss this at the meeting

Ok! Thanks for letting me know

**Comment by thai-truong** [comment]
Closed bc this issue is covered by another issue that already has a PR https://github.com/HKN-UCSD/hkn-server/issues/259
